Formula/a/apache-flink.rb

```diff
@@ -8,6 +8,11 @@ class ApacheFlink < Formula
   license "Apache-2.0"
   head "https://github.com/apache/flink.git", branch: "master"
 
+  livecheck do
+    url "https://flink.apache.org/downloads/"
+    regex(/href=.*?flink[._-]v?(\d+(?:\.\d+)+)-bin-scala_\d+\.\d+\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?flink[._-]v?(\d+(?:\.\d+)+)-bin[^"' >]*?\.t/i)
```

The file name format has varied a bit over the years, so there's something to be said about matching the `-bin` part of the suffix (which has been consistent) and glossing over the rest of it.